gui/main.js

```diff
@@ -85,12 +87,36 @@ const sendErrorToMainWindow = (msg) => {
       cancelId: 0,
       defaultId: 0
     }
+    trayWindow.hide()
     dialog.showMessageBox(null, options)
     desktop.stopSync()
       .then(() => desktop.removeConfig())
       .then(() => log.info('removed'))
       .then(() => trayWindow.doRestart())
       .catch((err) => log.error(err))
+    return // no notification
+  } else if (msg === 'Syncdir has been unlinked') {
```

Comment:
Maybe set some `#code` property on the error so we don't rely on the exact wording?

Reply:
in main.js errors are string, so we can't set an error.